test/e2e/dra/dra.go

```diff
@@ -105,6 +110,49 @@ var _ = ginkgo.Describe("[sig-node] DRA [Feature:DynamicResourceAllocation]", fu
 	})
 })
 
+// getExternalClaimTest generates test function for the external claim for certain allocation mode
+func genExternalClaimTest(ctx context.Context, f *framework.Framework, driver *Driver, allocationMode corev1.AllocationMode) func() {
+	return func() {
+		var b = newBuilder(f, driver)
+
+		ginkgo.It("registers plugin", func() {
```

Comment:
This test is independent of the allocation mode and doesn't need to be repeated.

Reply:
I tried to move builder creation and this test one level up, to the "kubelet" context. It didn't work out, may be because of https://github.com/onsi/ginkgo/issues/1022 ? Have I understood correctly that I'll have to create one more level of Context to make it working?

Reply:
I should probably do this inside BeforeEach, I guess.

Reply:
it didn't work :(

Reply:
should be fixed now (as far as I understood it)

Reply:
Didn't work for me, and I found the solution to be a bit too complex for my taste. Please have a look at the alternative solution in https://github.com/bart0sh/kubernetes/pull/4/files

Reply:
not sure I understand why it didn't work for you, but since you've changed it, it doesn't matter much.